config/3.3/ai-manager/Chart.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: v1
+name: aimanager33
+description: AIManager 3.3 Chart
+
+# A chart can be either an 'application' or a 'library' chart.
+#
+# Application charts are a collection of templates that can be packaged into versioned archives
+# to be deployed.
+#
+# Library charts provide useful utilities or functions for the chart developer. They're included as
+# a dependency of application charts to inject those utilities and functions into the rendering
+# pipeline. Library charts do not define any templates and therefore cannot be deployed.
+type: application
+
+# This is the chart version. This version number should be incremented each time you make changes
+# to the chart and its templates, including the app version.
+# Versions are expected to follow Semantic Versioning (https://semver.org/)
+version: 3.3
```

Comment:
This is the chart version, which is different from the appVersion we used.

For `appVersion`, we use `3.3` to represent the chart is for AIOps 3.3. For `version`, maybe we should use a different value. The reason for that is, it allows us to evolve the helm chart separately. For example, if the current chart version is `0.0.1`, then we can update the chart to fix whatever bug in a later version, `0.0.2`. To increase version number will be helpful for customers who have already installed using previous version, and they have option whether or not to update to a new version.

Reply:
Change the chart version to 0.0.1